src/component/search/label/NavbarSearch.tsx

```diff
@@ -108,7 +108,9 @@ export class NavbarSearch extends React.Component<NavbarSearchProps, NavbarSearc
 
         const clearIcon = (
             <InputGroupAddon addonType="append" id="search-reset" onClick={this.resetSearch} className="float-right">
-                <Button title={this.props.i18n("search.reset")} color="outline-danger" style={{zIndex : 5}}><FaTimes/></Button>
```

Comment:
Why is there a z-index set? I don't see any difference w and w/o it...

Reply:
Left-border of the button does not render red without it.

Reply:
I can see the border without any problems even when I remove the style. The addon button itself has z-index 2

Reply:
This is what I see without it 
![image](https://user-images.githubusercontent.com/1140626/97571261-0744f980-19e8-11eb-8df5-f45c5a66d625.png)

Chromium Version 86.0.4240.111 (Official Build) snap (64-bit)

Reply:
Ok, nevermind, I see it now, too (although it could be intentional in reactstrap).